crypto/platform/Cargo.toml

```
@@ -0,0 +1,26 @@
+[package]
+name = "mc-crypto-platform"
+description = "MobileCoin Crypto Platform Configurator Package"
+version = "0.1.0"
```

Our usual convention is to have all the `mc-*` packages in this repo have the same version. We might want that here
```suggestion
version = "2.0.0"
```